build errors with cargo install

I'm getting these build errors from `cargo install lolcrab`.

```
error[E0405]: cannot find trait `CommandFactory` in crate `clap`
  --> /Users/fdncred/.cargo/registry/src/github.com-1ecc6299db9ec823/lolcrab-0.1.3/src/rainbow_cmd.rs:26:17
   |
26 | #[derive(Debug, Parser)]
   |                 ^^^^^^ not found in `clap`
   |
   = note: this error originates in the derive macro `Parser` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0412]: cannot find type `Command` in crate `clap`
  --> /Users/fdncred/.cargo/registry/src/github.com-1ecc6299db9ec823/lolcrab-0.1.3/src/rainbow_cmd.rs:26:17
   |
26 | #[derive(Debug, Parser)]
   |                 ^^^^^^ not found in `clap`
   |
   = note: this error originates in the derive macro `Parser` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider importing this struct
   |
1  | use std::process::Command;
   |

error[E0433]: failed to resolve: could not find `Command` in `clap`
  --> /Users/fdncred/.cargo/registry/src/github.com-1ecc6299db9ec823/lolcrab-0.1.3/src/rainbow_cmd.rs:26:17
   |
26 | #[derive(Debug, Parser)]
   |                 ^^^^^^ not found in `clap`
   |
   = note: this error originates in the derive macro `Parser` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider importing this struct
   |
1  | use std::process::Command;
   |

Some errors have detailed explanations: E0405, E0412, E0433.
For more information about an error, try `rustc --explain E0405`.
error: could not compile `lolcrab` due to 3 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `lolcrab v0.1.3`, intermediate artifacts can be found at `/var/folders/m1/x0gp9jy51s146ttxtz7jlpxh0000gn/T/cargo-installmPBSrA`

Caused by:
  build failed


Updated 0 packages.
Failed to update lolcrab.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,7 +22,7 @@ default = ["clap"]
 
 [dependencies]
 bstr = "0.2.17"
-clap = { version = "= 3.0.5", features = ["derive"], optional = true }
+clap = { version = "= 3.1.8", features = ["derive"], optional = true }
 colorgrad = "0.5.0"
 fastrand = "1.6.0"
 noise = { version = "0.7.0", default-features = false }

